tests/tacacs/test_ro_disk.py

```diff
@@ -101,7 +101,13 @@ def do_reboot(duthost, localhost, duthosts):
 
 
 def post_reboot_healthcheck(duthost, localhost, duthosts, wait_time):
-    localhost.wait_for(host=duthost.mgmt_ip, port=22, state="started", delay=10, timeout=300)
+    timeout = 300
+    if duthost.get_facts().get("modular_chassis") and duthost.is_supervisor_node():
+        wait_time = max(wait_time, 900)
+        timeout = max(timeout, 600)
```

Comment:
Should we use same timeout for line cards?
In my observation, LC reboot could really near to 300s sometimes.
Same logic https://github.com/sonic-net/sonic-mgmt/blob/bc3989db213e419c3489ecc541b7d19965c631d5/tests/common/reboot.py#L259

Reply:
Done @yejianquan